data.sql

```diff
@@ -9,4 +9,23 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 ('Angemon','2005-6-7',1,true,-45),
 ('Boarmon','2005-6-12',7,true,20.4),
 ('Blossom','1998-10-13',3,true,17),
-('Ditto','2022-5-14',4,true,22);
\ No newline at end of file
+('Ditto','2022-5-14',4,true,22);
+
+INSERT INTO owners (full_name, age) VALUES 
+('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+INSERT INTO species  (name) VALUES 
+('Pokemon'),
+('Digimon');
+
+update animals set species_id = 2 where name like '%mon';
+update animals set species_id = 1 where name not like '%mon';
```

Comment:
> * [ ]  In this query it ends with `WHERE species_id IS NULL. ` It doesn't include %mon.


- [ ] As requested by the previous reviewer 🙃 , I think here is the best way to achieve so:

```sql
  UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;
```